File: example-microservice-java/src/main/java/com/example/java/customer/api/dto/CustomerDto.java
package com.example.java.customer.api.dto;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Setter
@Getter
@Builder
public class CustomerDto {

    private UUID id;
    @Schema(
            description = "Customer's first name",
            name = "firstName",
            type = "String",
            example = "George"
    )
    private String firstName;
    @Schema(
            description = "Customer's last name",
            name = "lastName",
            type = "String",
            example = "Clooney"
    )
    private String lastName;
    @Schema(
            description = "Customer's email address",
            name = "email",
            type = "String",
            example = "george.clooney@gmail.com"
    )
    private String email;

}

File: example-microservice-kotlin/src/main/kotlin/com/gooddata/kotlin/customer/api/dto/CustomerDto.kt
package com.example.kotlin.customer.api.dto

import io.swagger.v3.oas.annotations.media.Schema
import java.util.*

data class CustomerDto(

    val id: UUID? = null,
    @Schema(
        description = "Customer's first name",
        name = "firstName",
        type = "String",
        example = "George"
    )
    val firstName: String?,
    @Schema(
        description = "Customer's last name",
        name = "lastName",
        type = "String",
        example = "Clooney"
    )
    val lastName: String?,
    @Schema(
        description = "Customer's email address",
        name = "email",
        type = "String",
        example = "george.clooney@gmail.com"
    )
    val email: String?

)

{

    constructor(): this(null, "", "", "")

}